parcels/field.py

```diff
@@ -151,13 +170,17 @@ class Field(object):
 
     def __init__(self, name, data, lon=None, lat=None, depth=None, time=None, grid=None,
                  transpose=False, vmin=None, vmax=None, time_origin=0,
-                 interp_method='linear', allow_time_extrapolation=None, time_periodic=False):
+                 interp_method='linear', allow_time_extrapolation=None, time_periodic=False, mesh='flat'):
         self.name = name
+        if self.name == 'UV':
+            self.phantom = True
```

Comment:
Indeed, it was called `phantom` since it is an un-complete field. 
When I did it, I thought we might use such field for another reason later, it is why I didn't give this flag a so explicit name. 
But you are right, let's advance one step at a time. So for the moment, phantom is used only for 'UV', so I simply remove it, and check if self.name == 'UV' instead of self.phantom